api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimService.java

```diff
@@ -171,6 +172,7 @@ private Claim buildClaim(ClaimStatus claimStatus, ClaimRequest claimRequest, Eli
                 .webUIVersion(claimRequest.getWebUIVersion())
                 .initialIdentityAndEligibilityResponse(decision.getIdentityAndEligibilityResponse())
                 .currentIdentityAndEligibilityResponse(decision.getIdentityAndEligibilityResponse())
+                .eligibilityOverrideOutcome(claimRequest.getEligibilityOverrideOutcome())
```

Comment:
An open question: do we need to store this after the override has been applied, is this here just for record keeping? Feels like that might be better stored in the CRM system (and yes I do mean the CRM system).

Reply:
Yes, it will need to be applied in every payment cycle - so it needs to be stored with the claim.